tests/integration/test_keeper_disks/test.py

```diff
@@ -163,12 +165,22 @@ def test_logs_with_disks(started_cluster):
             look_behind_lines=2000,
         )
 
+        def get_local_log_files():
+            local_log_files = get_local_logs(node_logs)
+            start_time = time.time()
+            while len(local_log_files) != 1:
+                assert (
```

Comment:
Maybe also consider adding a log line to list all the logs files, so we can understand the situation more if there will be any: `logging.debug(f"Local log files: {local_log_files}")